configs/98netflix

```diff
@@ -0,0 +1,6 @@
+#!/system/bin/sh
```

Comment:
first: this device ain't going to get any official cm
second: except s2 and 1 smdk common have this script in it, there is no indication anywhere that told device should support netflix to get official support